static/js/pages/receipts.js

```diff
@@ -65,4 +68,25 @@ $(document).ready(function() {
       pagingType: 'simple'
     }
   );
+
```

Comment:
Should this be moved to a different file? Will be used by /disbursements as well.